modules/serverless-rn-1-8-0.adoc

```diff
@@ -9,14 +9,73 @@
 [id="new-features-1-8-0_{context}"]
 == New features
 
-* {ServerlessProductName} now uses Knative Serving 0.x
-* {ServerlessProductName} now uses Knative Serving Operator 0.x
-* {ServerlessProductName} now uses Knative `kn` CLI 0.x
-* {ServerlessProductName} uses Knative Eventing 0.x
-* {ServerlessProductName} now uses Knative Eventing Operator 0.x
+* {ServerlessProductName} now uses Knative Serving 0.14.1.
+* {ServerlessProductName} now uses Knative Serving Operator 0.14.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.14.0.
+* {ServerlessProductName} uses Knative Eventing 0.14.2.
+* {ServerlessProductName} now uses Knative Eventing Operator 0.14.0.
+* {ServerlessProductName} now uses Kourier 0.14.1.
 
-[id="fixed-issues-1-8-0_{context}"]
-== Fixed issues
+// [id="fixed-issues-1-8-0_{context}"]
+// == Fixed issues
 
 [id="known-issues-1-8-0_{context}"]
 == Known issues
+* Knative Serving has a fixed, minimum CPU request of 25m for `queue-proxy`.
+If your cluster has any value set that conflicts with this, for example, if you have set a minimum CPU request for `defaultRequest` of more than `25m`, the Knative Service will fail to deploy.
+As a work-around, you can configure `resourcePercentage` individually for your Knative Services.
++
+.Example resourcePercentage configuration
++
+[source.yml]
+----
+spec:
+  template:
+    metadata:
+      annotations:
+        queue.sidecar.serving.knative.dev/resourcePercentage: "10"
+----
++
+`queue.sidecar.serving.knative.dev/resourcePercentage` is the percentage of user container resources to be used for `queue-proxy`. This can be between a range of 0.1 - 100.
+
+* On OpenShift 4.5 RC-6, deploying a Knative Service with traffic distribution will show an invalid URL for the general service address in the *Developer Perspective* of the web console.
```

Comment:
This should be on OCP 4.5 and 4.6. I would also add that valid address can be seen in other places like YAML and CLI.